pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -250,13 +250,13 @@ ARG PT_TORCHDATA_URL
 RUN /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION} \
   torchvision torchaudio torchtext cpuonly \
   --override-channels \
-  -c https://aws-ml-conda-pre-prod-ec2.s3.us-west-2.amazonaws.com \
+  -c https://aws-ml-conda-pre-prod-sm.s3.us-west-2.amazonaws.com \
```

Comment:
Is there a reason we are using SM in the EC2 recipe?

Reply:
I think this comes from the bug fixes.

Reply:
This is strange. Can we confirmed with @roywei?

Reply:
I see its ec2 here - https://github.com/aws/deep-learning-containers/pull/2899/files#diff-bc0b77b02ee5079e588e1fd8714585bee92e4159c585a980f2f576beffbc6228L271

Reply:
this is coming from bug fixes. Debugger patch is causing issues with SMP and we disabled it. Hence using EC2 binaries in SM DLC for now.

Reply:
@roywei I see EC2 in SM, and SM in EC2. Is that intentional?

Reply:
sorry for the confusion, for this specific line, it should be ec2, it should all be ec2 binaries for PT 2.0 release, as we disabled debugger and Rubik patch is no longer needed